agent/spec/lib/kontena/workers/volumes/volume_manager_spec.rb

```diff
@@ -149,6 +149,11 @@
       }.to raise_error(Kontena::Workers::Volumes::VolumeManager::DriverMismatchError)
     end
 
+    it 'return true if volume exists with different plugin version' do
```

Comment:
Also needs a spec for the problematic `expected: 'rexray/s3fs', actual: 'rexray/s3fs:latest' case?